workflow/apm/apm-overview.mdx

```diff
@@ -0,0 +1,26 @@
+---
+title: Intro to APM
+---
+
+The Application Performance Monitoring (APM) dahshboard is a unified dashboard that provides real-time insights into the performance of your applications. Whether instrumenting your APMs with the Infrastructure (Infra) Agent or Serverless, all of your APM data is displayed within this section of the Middleware.

```

Comment:
We should call this a section rather than a dashboard, because the APM page itself is multiple views and dashboards. "The Application Performance Monitoring (APM) section is a collection of views and dashboards that provide real-time insights..."